tests/test_playbooks/users_role.yml

```diff
@@ -25,6 +31,7 @@
     server_url: "{{ foreman_server_url }}"
     validate_certs: "{{ foreman_validate_certs }}"
     organization: "Test Organization"
+    location: Minga
```

Comment:
;)
I believe that location is where your server is located. But you should add `tasks/location.yml` to the included tasks and use, i guess, "Test Location".

Reply:
:-)